jenkins/Jenkinsfile

```diff
@@ -194,12 +194,14 @@ pipeline {
           dir('mobilecoind/strategies') {
             sh 'pip3 install -r requirements.txt'
             sh '''
-              python3 -m grpc_tools.protoc -I../../consensus/api/proto \
-                --python_out=. ../../consensus/api/proto/external.proto
+              python3 -m grpc_tools.protoc \
+                -I$WORKSPACE/consensus/api/proto -I$WORKSPACE/api/proto \
```

Comment:
Since this only compiles `external.proto` it doesn't need `-I$WORKSPACE/consensus/api/proto`

Reply:
👍